src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/ConfirmEmployer.cshtml

```diff
@@ -0,0 +1,66 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.ConfirmEmployerViewModel
+
+@{
+    ViewBag.Title = "Confirm Employer";
+}
+
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper" role="main">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <partial name="_ValidationSummary" />
```

Comment:
The link in the validation summary if the user fails to answer the question does not jump the user to the appropriate anchor.